README.md

```diff
@@ -1,26 +1,16 @@
-# README
+# Repository Mirroring Tool
 ![Kartoha](https://travis-ci.org/SUSE/rmt.svg?branch=master)
 
+This tool allows you to mirror RPM update repositories in your own private network. Registered organization credentials are required to mirror SUSE repositories.
```

Comment:
Wording: `update repositories`. Please drop the word `update`.
There are Pool and Update repositories, and RMT can mirror them both! :grimacing: